src/olympia/devhub/views.py

```diff
@@ -126,9 +125,7 @@ def index(request):
         recent_addons = request.user.addons.all().order_by('-modified')[:3]
         ctx['recent_addons'] = []
         for addon in recent_addons:
-            ctx['recent_addons'].append(
-                {'addon': addon, 'position': get_position(addon)}
-            )
+            ctx['recent_addons'].append({'addon': addon})
```

Comment:
If you're up for going a little further, I just noticed this is now useless... 

The whole loop could be completely removed, and `recent_addons` assigned to `ctx['recent_addons']` directly. Then, in the template (`src/olympia/devhub/templates/devhub/new-landing/components/my-addons.html`), you would modify the loop over `recent_addons` to do `{% for addon in recent_addons %}`, removing the `{% set addon = item.addon %}` bit.

Does that make sense ?

Reply:
Oh okay, yes I got it.
Will make the changes.